build.proj

```diff
@@ -15,4 +15,16 @@
   <Target Name="Clean">
     <Delete Files="$(BinDir)mscorlib.*" />
   </Target>
+
+  <!-- Restore the Microsoft.NETCore.Platforms package required for packaging validation -->
+  <ItemGroup>
+    <ProjectJSONToRestore Include="$(SourceDir)/.nuget/init/project.json" />
+  </ItemGroup>
+
+  <Target Name="RestoreNETCorePlatforms" AfterTargets="Build">
+    <!-- Copy to bin output folder to avoid generating project.lock.json and pollute the source folders -->
```

Comment:
We should just add project.lock.json to gitignore, IMHO.  That's what we do in CoreFX.